gemma-core/src/main/java/ubic/gemma/core/genome/gene/service/GeneSearchServiceImpl.java

```diff
@@ -166,13 +166,14 @@ public Collection<SearchResultDisplayObject> searchGenesAndGeneGroups( String qu
                 .query( query )
                 .resultType( Gene.class )
                 .resultType( GeneSet.class )
+                .quickSearch( true )
```

Comment:
No need for a true here, maybe even have a quickSearch() builder instead and make the field final?